### PR TITLE
annotate secrets with path and version

### PR DESCRIPTION
**Pull request by tparikh**
Signed-off-by: Tejas Parikh <tparikh@redhat.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,7 +5,7 @@
 .tox
 .pytest_cache
 .eggs
-
+.vscode/*
```
Comment (maorfr) [inline .gitignore]: this should go in your global git ignore so you don't have to repeat it in every project

File: reconcile/openshift_resources.py [hunk 2]
```diff
@@ -354,6 +354,8 @@ def fetch_openshift_resource(resource, parent):
         labels = {} if rl is None else json.loads(rl)
         ra = resource['annotations']
         annotations = {} if ra is None else json.loads(ra)
+        annotations['vault.secret.path'] = '' if path is None else path
+        annotations['vault.secret.version'] = '' if version is None else version
```
Comment (maorfr) [inline reconcile/openshift_resources.py]: line too long

**Comment by tparikh** [comment]
@maorfr I did run it locally but this can wait until you have verified as this will update all existing secrets with these annotations. Please merge after you have verified it tomorrow.

**Comment by maorfr** [comment]
i ran it locally, and it seems that these annotations were not actually applied.

**Pull request closed without merging**